radixdlt/src/main/java/com/radixdlt/syncer/SyncedEpochExecutor.java

```diff
@@ -49,8 +49,14 @@
 		void sendLocalSyncRequest(LocalSyncRequest request);
 	}
 
+	public interface Executor {
```

Comment:
Probably should use a different name for this class, given this already exists in the standard JDK libs.